handlers/config.go

```diff
@@ -50,6 +51,7 @@ type PublicConfig struct {
 	AuthStrategy         string                        `json:"authStrategy,omitempty"`
 	AmbientEnabled       bool                          `json:"ambientEnabled,omitempty"`
 	Clusters             map[string]models.KubeCluster `json:"clusters,omitempty"`
+	ControlPlaneClusters []string                      `json:"controlPlaneClusters,omitempty"`
```

Comment:
I don't think this belongs on the config because it's not a configuration setting. We just need a way to know if the user has access to the istio system namespace without necessarily calling `/api/mesh`? Maybe it can be added to the session info redux state? Actually as part of the post login actions I think we already have a list of all the namespaces. This can be some redux state derived from that list that just checks for the presence of the istio system namespace for each cluster.